AsyncDisplayKit/ASVideoNode.h

```diff
@@ -0,0 +1,34 @@
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+typedef NS_ENUM(NSUInteger, ASVideoGravity) {
+  ASVideoGravityResizeAspect,
+  ASVideoGravityResizeAspectFill,
+  ASVideoGravityResize
+};
+
+@protocol ASVideoNodeDelegate;
+
+@interface ASVideoNode : ASControlNode<_ASDisplayLayerDelegate>
+@property (atomic, strong, readwrite) AVAsset *asset;
+@property (atomic, strong, readonly) AVPlayer *player;
+@property (atomic, strong, readonly) AVPlayerItem *currentItem;
+@property (nonatomic, assign, readwrite) BOOL shouldAutoplay;
+@property (atomic) ASVideoGravity gravity;
```

Comment:
We should probably just use the AV type for this (even though they are NSStrings, do they have any other type name for them?)  I suppose it is unlikely they'll introduce more, but still it would be best to minimize any new APIs unless they are easier for folks to use.  Would need to add a comment referencing the intended type / maybe just specify "// values from AVAsset.h gravity" or something, especially if it is just NSString.